### PR TITLE
feature/CORE-665/fix-choice-layout

### PR DESCRIPTION
**Pull request by bulatgab**
voto5 PR: https://bitbucket.org/voto/voto5/pull-requests/3399

### CODE REVIEW

File: src/components/common/ExpressionInput.vue [hunk 1]
```diff
@@ -4,28 +4,23 @@
...
-        class="cloned-auto-suggest-content flex-row" />
+      <textarea
+        ref="input"
+        v-model="expression"
+        :class="['form-control', {'is-invalid': isInvalid}]"
+        :disabled="disabled"
+        :rows="rows"
+        :placeholder="placeholder"
+        @click="handleClick()"
+        @input="$emit('input', $event.target.value)" />
```
Comment (bulatgab) [inline src/components/common/ExpressionInput.vue:21]: 1. I have undone the layout changes from CORE-461.

File: src/components/common/ExpressionInput.vue [hunk 2]
```diff
@@ -4,28 +4,23 @@
...
     </div>
+    <div ref="suggest" />
```
Comment (bulatgab) [inline src/components/common/ExpressionInput.vue:23]: 2. I have moved out the autosuggest's  'anchor' element to be below the input.

File: src/components/common/ExpressionInput.vue [hunk 3]
```diff
@@ -43,7 +38,8 @@ import {getSuggestions} from '@/lib/suggestions'
 
 interface IAutoSuggest {
   dropdown: {
-    dropdown: HTMLElement,
+    dropdown: HTMLDivElement,
+    dropdownContent: HTMLUListElement,
```
Comment (bulatgab) [inline src/components/common/ExpressionInput.vue:42]: slightly better TS

File: src/views/InteractionDesigner.vue [hunk 4]
```diff
@@ -3,7 +3,9 @@
     v-if="activeFlow"
     ref="interaction-designer-contents"
```
Comment (bulatgab) [inline src/views/InteractionDesigner.vue:4]: in this file: ESLint fixes + import overridden css

**Review by object-required — APPROVED** [review]

**Pull request merged**